README.md

```
@@ -1,337 +1,71 @@
-    value = false;
+A estrutura é organizada da seguinte forma:
 
-## Palavras-chave
+* **Camada de Dados (Data):** É o coração da lógica do aplicativo. Ela contém:
+    * **`DataSources`**: O ponto de contato direto com a API. São responsáveis por executar as requisições HTTP e entregar os dados brutos para os Repositórios.
+    * **`Repositories`**: A única fonte de verdade para os `Controllers`. Servem como uma camada de abstração que centraliza e gerencia o acesso aos `DataSources`.
+    * **`Models`**: Classes que fazem o parsing do JSON da API para objetos Dart.
+    * **`Controllers (GetX)`**: Classes que gerenciam o estado da UI, a lógica de interação e a orquestração das chamadas aos repositórios.
 
-Se você quer que uma variável nunca seja alterada deve-se usar final ou const, em vez de usar var ou somente o tipo. Uma variável definida como final pode ter seu valor definido apenas uma vez, já uma variável const é uma constante de tempo de compilação. (Uma variável const é implicitamente final)
+* **Camada de Apresentação (Presentation):** Responsável exclusivamente pela parte visual do aplicativo.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)